includes/Admin/Admin_Page.php

```diff
@@ -150,16 +150,20 @@ private function get_available_plugins() {
 	 * Renders the "Plugin Check" page.
 	 *
 	 * @since n.e.x.t
+	 *
+	 * @SuppressWarnings(PHPMD.UnusedLocalVariable)
 	 */
 	public function render_page() {
-		global $available_plugins, $selected_plugin_basename, $categories;
-
 		$available_plugins = $this->get_available_plugins();
 
 		$selected_plugin_basename = filter_input( INPUT_GET, 'plugin', FILTER_SANITIZE_FULL_SPECIAL_CHARS );
 
 		$categories = Check_Categories::get_categories();
 
+		// Get user settings for category preferences and set a default value to check all categories by default.
+		$user_settings = get_user_setting( 'plugin_check_category_preferences', 'all_categories' );
+		$user_settings = explode( '__', $user_settings );
```

Comment:
See my other comment, we can simplify this to avoid the custom `all_categories` value:
```suggestion
		$user_settings = 'all_categories' === $user_settings ? $categories : explode( '__', $user_settings );
```

On another note: Maybe use a more descriptive variable name? `$user_settings` is what's used under the hood to store the value, but it doesn't describe well what the variable contains. Maybe `$user_enabled_categories` or `$user_checked_categories`?